containers/Connector/Connector.tsx

```diff
@@ -65,22 +66,29 @@ const useConnector = () => {
 
 	useEffect(() => {
 		const init = async () => {
-			// TODO: need to verify we support the network
 			const networkId = await getDefaultNetworkId();
 
-			const provider = loadProvider({
+			const _provider = loadProvider({
```

Comment:
Internal variable syntax probably more used in solidity than here.

Reply:
`provider` is already defined in the outer scope which is why I changed it to `_provider`. WDYT?

Reply:
hmm interesting it was working before, is `provider` coming from `synthetix` ?